doc/conf.py

```diff
@@ -17,7 +17,12 @@
 import os.path as op
 from datetime import date
 
-import sphinxgallery
+try:
+    import sphinx_gallery as sg
+    extension = 'sphinx_gallery.gen_gallery'
+except:
```

Comment:
`except ImportError` or at least `except Exception` (what you have now will even catch `SystemExit`)